test/rls/guardrail-proof.mjs

```
@@ -0,0 +1,210 @@
+  // fails, but still exit non-zero if any cleanup failed - a silently-
+  // swallowed cleanup failure would leave stale users/events behind while
+  // reporting success.
+  const results = await Promise.allSettled(
+    [actorA, actorB].filter((actor) => actor !== undefined).map((actor) => deleteTestActor(actor)),
+  );
+  const failures = results.filter((result) => result.status === 'rejected');
+  if (failures.length > 0) {
+    const messages = failures.map((failure) =>
+      failure.reason instanceof Error ? failure.reason.message : String(failure.reason),
+    );
+    throw new Error(`test actor cleanup failed:\n${messages.join('\n')}`);
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)